packages/icons/docs/stories/rainbowIconItem.js

```diff
@@ -80,7 +80,6 @@ const RainbowIconItem = (props) => {
             <StyledButton
                 id={iconBoxId}
                 ref={triggerRef}
-                onMouseEnter={openOverlay}
```

Comment:
You have to check this component well and eliminate all the necessary logic, you only have to leave what is related to the isClickTooltip

Reply:
@LeandroTorresSicilia this component is based on the HelpText, if we change the behavior, everything related to the hover and the setTimeout is not necessary.